lib/routes/backstage.js

```diff
@@ -7,10 +7,12 @@ class BackstageController extends Controller {
     constructor(config) {
         super();
 
+        this.logger = config.getLogger('backstage.js');
+
         if (config.serverMetrics) {
-            this.get('/prometheus', (req, res) => {
+            this.get('/prometheus', async (req, res) => {
```

Comment:
`express` is horrible with async-await, so should wrap in `try-catch` and pass any errors to `next`

Reply:
I agree that this isn't ideal, but practically all controllers in the app use async handlers in this way.

Given that it's like this everywhere, I don't want to create a new incantation for setting up async handlers for this particular handler, which will never fail (i.e., isn't dependent on i/o).

Reply:
I think we are fine with this. We are planning to replace all of express in next major.